app/i18n/locales/terms-of-use/itn/cs-CZ.md

```diff
@@ -1,9 +0,0 @@
-## 12. Delegation and Staking
```

Comment:
I think these should also be automatically updated by CrowdIn so modifying these files directly will cause merge conflicts

Reply:
@SebastienGllmt yeah I need to change the configuration from Crowdin first -- although stop tracking the changes on those files won't delete them afaik